src/test/java/duke/StorageTest.java

```diff
@@ -0,0 +1,89 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.api.AfterAll;
+
+import java.io.File;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.io.FileWriter;
+
+public class StorageTest {
+
+    @Test
+    public void testStorageCreation() {
```

Comment:
Maybe consider adopting the 3 part test method naming convention?
`featureUnderTest_testScenario_expectedBehavior()`